### PR TITLE
Create corretto11.rb

### PR DESCRIPTION
**Pull request by anvar**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

This is a copy of the previous version of `corretto.rb`, which was Corretto 11 based, as the new one is Corretto 15 based.

After making all changes to a cask, verify:

- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask-versions/pulls) for the same update.
- [ ] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-versions/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/corretto11.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask "corretto11" do
+  version "11.0.8.10.1"
+  sha256 "d87c677c3f2d5b323112aa8e028cd4dbaad0e37fb378ffac87f7e447996476cb"
+
+  url "https://corretto.aws/downloads/resources/#{version.sub(/-\d+/, "")}/amazon-corretto-#{version}-macosx-x64.pkg"
```
Comment (vitorgalvao) [inline Casks/corretto11.rb:5]: `#{version.sub(/-\d+/, "")}` is an empty string.

**Review by vitorgalvao — DISMISSED** [review]

**Comment by miccal** [comment]
https://github.com/Homebrew/homebrew-cask/pull/90927#issuecomment-710766651

File: Casks/corretto11.rb (hunk 1 above)
Comment (karl82) [reply Casks/corretto11.rb:5]: It's not empty an empty string. It won't substitute `version`. The full URL from corretto releases is `https://corretto.aws/downloads/resources/11.0.8.10.1/amazon-corretto-11.0.8.10.1-macosx-x64.pkg`.

Can you please approve the change?

Comment (vitorgalvao) [reply Casks/corretto11.rb:5]: > It's not an empty string.

Bad phrasing on my part. I meant to say the substitution is essentially empty.

> It won't substitute anything in `version`.

And that’s the issue, the `sub` is useless.

> Can you please approve the change?

Not until that’s fixed. It needs to be changed to `#{version}`.

Comment (karl82) [reply Casks/corretto11.rb:5]: I guess this is copy&paste from original `corretto.rb`

https://github.com/Homebrew/homebrew-cask/pull/90927/commits/a439ae392044c70df3d9d17047b70e0786fff14f#diff-3a23d69965c0dc050c7ef26037b83f2b740734e925ed7531d3be3cc3513890c5L5

Comment (vitorgalvao) [reply Casks/corretto11.rb:5]: > I guess this is copy&paste from original `corretto.rb`

Which doesn’t make it any less wrong. Nobody’s saying the error was introduced by you (wouldn’t matter even if it were); but it makes no sense to keep the error.

Comment (karl82) [reply Casks/corretto11.rb:5]: > Nobody’s saying the error was introduced by you (wouldn’t matter even if it were); but it makes no sense to keep the error.

I'm not arguing. Just trying to figure how to update current PR with fix

Comment (vitorgalvao) [reply Casks/corretto11.rb:5]:
```suggestion
  url "https://corretto.aws/downloads/resources/#{version}/amazon-corretto-#{version}-macosx-x64.pkg"
```

Comment (karl82) [reply Casks/corretto11.rb:5]: I have this change locally. I haven't been using `gh` and contributing into github repos for a while...

**Comment by miccal** [comment]
Thank you @anvar and @karl82.

**Review by miccal — APPROVED** [review]

**Pull request merged**